public/app/features/manage-dashboards/state/actions.ts

```diff
@@ -182,6 +233,22 @@ const getDataSourceOptions = (input: { pluginId: string; pluginName: string }, i
   }
 };
 
+const getDataSourceDescription = (input: { usage?: InputUsage }): string | undefined => {
+  if (!input.usage) {
+    return undefined;
+  }
+
+  if (input.usage.libraryPanels) {
+    const libPanelNames = input.usage.libraryPanels.reduce(
+      (acc: string, libPanel, index) => (index === 0 ? libPanel.name : `${acc}, ${libPanel.name}`),
+      ''
+    );
+    return `List of affected library panels: ${libPanelNames}`;
+  }
+
+  return undefined;
+};
+
```

Comment:
It shows a description above the input in order to explicitly enum the **new library panels** that need the specific datasource